stable/sonarqube/templates/ingress.yaml

```
@@ -1,7 +1,7 @@
 {{- if .Values.ingress.enabled -}}
 {{- $serviceName := include "sonarqube.fullname" . -}}
 {{- $servicePort := .Values.service.externalPort -}}
-apiVersion: extensions/v1beta1
+apiVersion: networking.k8s.io/v1
```

How about this to make it compatible with versions <= k8s 1.13?

```suggestion
{{ if .Capabilities.APIVersions.Has "networking.k8s.io/v1" }}
apiVersion: networking.k8s.io/v1
{{ else }}
apiVersion: extensions/v1beta1
{{ end -}}
```

---

I don't think it's required, since this was apparently available since v1.8 (unless I'm misreading [the changelog](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG-1.14.md#deprecations)).

Can you confirm it's a problem on k8s versions <= 1.13 as-is?

Thanks for your quick feedback!